test/cluster-store.js

```diff
@@ -2,7 +2,7 @@ var cluster = require('cluster');
 var expect = require('chai').expect;
 var ioServer = require('socket.io');
 var ioClient = require('socket.io-client');
-var debug = require('debug')('test');
+var debug = require('debug')('strong-cluster-socket.io-store:test');
```

Comment:
I prefer to use `test`, as it's much shorter to type. You are not likely to run tests from multiple modules, are you? Plus I find it useful to enable only logs from the production code, which requires even more typing with the debug schema you are proposing here.